pilot.go

```diff
@@ -87,9 +87,11 @@ func (c *chanController) OpenChannel(target *btcec.PublicKey,
 
 	// TODO(halseth): make configurable?
 	minHtlc := lnwire.NewMSatFromSatoshis(1)
+	minConfs := uint32(0)
```

Comment:
make autopilot configuration option that defaults to zero?

Reply:
Added!